lib/QubitCsvImport.class.php

```diff
@@ -135,14 +143,14 @@ public function import($csvFile, $type = null, $csvOrigFileName = null)
         $commandSkipUnmatched,
         $commandSkipMatched,
         $commandUser,
-        escapeshellarg($csvOrigFileName),
+        $commandSourceName,
         escapeshellarg($this->parent->slug),
         escapeshellarg($transformedFile ? $transformedFile : $csvFile));
     }
     else
     {
       // Example: php symfony csv:import /tmp/foobar
-      $command = sprintf('php %s %s %s %s %s %s %s --quiet %s --source-name=%s %s',
+      $command = sprintf('php %s %s %s %s %s %s %s %s %s --quiet %s',
```

Comment:
Having a whole separate `$command` assignment for the `--default-parent-slug` case seems like unnecessary repetition that could lead to mistakes where a new option is added to one case but not the other.  Can this if/else be combined into a single statement?